doc/v2/documentation_data_format.rst

```diff
@@ -816,13 +811,10 @@ Detailed field description
     - *parameterScaleNormal*: Gaussian prior on original parameter scale
     - *parameterScaleLaplace*: Laplace prior on original parameter scale
 
-- ``initializationPriorParameters`` [STRING, OPTIONAL]
+- ``priorParameters`` [STRING, OPTIONAL]
 
-  Prior parameters used for sampling of initial points for estimation,
-  separated by a semicolon. Defaults to ``lowerBound;upperBound``.
-  The parameters are expected to be in linear scale except for the
-  ``parameterScale`` priors, where the prior parameters are expected to be
-  in parameter scale.
+  Prior parameters used for the :ref:`MAP objective function and for Bayesian inference <v2_objective_function>`.
+  ``priorParameters`` is required if ``prior`` is non-empty. If ``prior`` is specified for any other parameter, and ``prior`` is  empty for the given parameter, this defaults to ``lowerBound;upperBound``.
```

Comment:
Moved to the `prior` description, since it seems clearer to me that way.
```suggestion
  ``priorParameters`` is required if ``prior`` is non-empty.
```